lib/iris/tests/unit/fileformats/nc_load_rules/actions/test__time_coords.py

```diff
@@ -359,31 +364,17 @@ def test_aux_not_in_phenom_coords(self):
         self.check_result(result, "missing")
 
     def test_aux_fails_typeident(self):
-        # The coord variable is identified as a CFAuxiliaryCoordinate by cf.py,
-        # but having the wrong units causes it to fail the 'is_time' or
-        # 'is_period' test, so the 'provides_coord' rule fails to trigger.
-        # Rules Triggered:
-        #     001 : fc_default
-        #     002 : fc_build_auxiliary_coordinate
-        # Again, though it builds as a 'miscellaneous' rather than a recognised
-        # specific coord type, it makes no practical difference.
-        result = self.run_testcase(
-            coord_dim_name="dim_renamed",
-            dimname="dim_renamed",
-            in_phenomvar_coords=True,
-            units="1",
-        )
-        self.check_result(result, "aux")
-
-    def test_aux_no_coordsref(self):
```

Comment:
NOTE: removed this testcase, as it seems to be a mistaken leftover -- identical to the previous one. 
From the name, the intended testcase was probably the same as the existing `test_aux_not_in_phenom_coords`.